composer.json

```
@@ -23,7 +23,7 @@
     "require": {
         "php": "^5.4 | ^7 | ^8",
         "ext-json": "*",
-        "paragonie/random_compat": "^1 | ^2 | 9.99.99",
+        "paragonie/random_compat": "^1 | ^2 | ^9.99.99",
```

It's probably better to require `>= 1` here rather than explicit major versions, in line with packages like defuse/php-encryption (which requires `>= 2`). v3 or v10 are unlikely, but never say never ;)

```suggestion
        "paragonie/random_compat": ">= 1",
```

---

Using semver with `>= 1`, the problem is that v3 or v10 are allowed to break BC with v1 and v2, which could cause more problems. In the highly unlikely event there is a v3 or higher in paragonie/random_compat, we'll update this library. 😄

---

Sounds good :)